jira_docugen.tex

```diff
@@ -0,0 +1,971 @@
+filesystem:\\[2\baselineskip]\hspace*{0.333em} ~ setup git\_lfs\\
+\hspace*{0.333em} ~ git clone
+https://github.com/lsst/ap\_verify\_hits2015.git\\[2\baselineskip]\hspace*{0.333em}
+~ export AP\_VERIFY\_HITS2015\_DIR=\$PWD/ap\_verify\_hits2015 cd
+\$AP\_VERIFY\_HITS2015\_DIR\\
+\hspace*{0.333em} ~ setup -r .\\
+\hspace*{0.333em} ~ cd-\\
+\hspace*{0.333em} ~\\
+\hspace*{0.333em} ~ setup obs\_decam\\
+\hspace*{0.333em} ~ git clone
+https://github.com/lsst-dm/ap\_association\\
+\hspace*{0.333em} ~ cd ap\_association\\
+\hspace*{0.333em} ~ setup -k -r .\\
+\hspace*{0.333em} ~ scons\\
+\hspace*{0.333em} ~ cd-\\
+\hspace*{0.333em} ~\\
+\hspace*{0.333em} ~ git clone https://github.com/lsst-dm/ap\_pipe\\
+\hspace*{0.333em} ~ cd ap\_pipe\\
+\hspace*{0.333em} ~ setup -k -r .\\
+\hspace*{0.333em} ~ scons\\
+\hspace*{0.333em} ~ cd-\\
+\hspace*{0.333em} ~\\
+\hspace*{0.333em} ~ git clone https://github.com/lsst-dm/ap\_verify\\
+\hspace*{0.333em} ~ cd ap\_verify\\
+\hspace*{0.333em} ~ setup -k -r .\\
+\hspace*{0.333em} ~ scons\\
+\hspace*{0.333em} ~ cd-\\[2\baselineskip]and any errors or failures
+reported.\\[2\baselineskip]
+
+\hypertarget{lvv-t18---ag-00-05-alert-generation-produces-required-data-products}{\subsection{\texorpdfstring{\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T18}{LVV-T18}
+- AG-00-05: Alert Generation Produces Required Data
+Products}{LVV-T18 - AG-00-05: Alert Generation Produces Required Data Products}}\label{lvv-t18---ag-00-05-alert-generation-produces-required-data-products}}
+
+\begin{longtable}[]{@{}llllll@{}}
+\toprule
+Version & Status & Priority & Verification Type & Critical Event &
+Owner\tabularnewline
+\midrule
+\endhead
+1 & Approved & Normal & Test & False & Eric Bellm\tabularnewline
+\bottomrule
+\end{longtable}
+
+\subsubsection{Test Items}\label{test-items-1}
+
+This test will check that the basic data products produced by Alert
+Generation are generated by execution of the science payload.\\
+These products will include:
+
+\begin{itemize}
+\tightlist
+\item
+  Processed visit images (PVIs; DMS-REQ-0069); • Difference Exposures
+  (DMS-REQ-0010);
+\item
+  DIASource catalogs (DMS-REQ-0269);
+\item
+  DIAObject catalogs (DMS-REQ-0271);
+\end{itemize}
+
+\subsubsection{Requirements}\label{requirements-1}
+
+\begin{itemize}
+\tightlist
+\item
+  \href{https://jira.lsstcorp.org/browse/LVV-29}{LVV-29} -
+  DMS-REQ-0069-V-01: Processed Visit Images
+\item
+  \href{https://jira.lsstcorp.org/browse/LVV-7}{LVV-7} -
+  DMS-REQ-0010-V-01: Difference Exposures
+\item
+  \href{https://jira.lsstcorp.org/browse/LVV-100}{LVV-100} -
+  DMS-REQ-0269-V-01: DIASource Catalog
+\item
+  \href{https://jira.lsstcorp.org/browse/LVV-102}{LVV-102} -
+  DMS-REQ-0271-V-01: DIAObject Catalog
+\end{itemize}
+
+\subsubsection{Precondition}\label{precondition-1}
+
+- Input specification\\[2\baselineskip]A complete processing of the
+DECam ``HiTS'' dataset, as defined at https://dmtn-039.lsst. io/ and
```

Comment:
Not sure why this extra space ended up in LVV_T18 Precondition section. I have corrected it in jira. It may have been already present in the previous issue of LDM-533.